src/components/link/link.test.tsx

```diff
@@ -0,0 +1,570 @@
+// We've chosen to ignore this for the whole file as this warning fires 15 times.
+// It warns that `... anchors are primarily expected to navigate. Use the button element instead.`
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+import Link from "./link.component";
+import MenuContext from "../menu/menu.context";
+import { baseTheme } from "../../style/themes";
+
+describe("If `isSkipLink` provided", () => {
+  it("should render `Skip to main content` text inside of Link", () => {
```

Comment:
**comment(non-blocking)**: I assume we need the below style tests for coverage reasons too. If not, could we utilise Playwright or Chromatic? This would offer the benefit of increased confidence in our style tests, as the component would be rendered in an actual browser rather than with JSDOM.

Reply:
Yes we need these for coverage. Something to consider once we have the shared coverage completed though.